### ISSUE TITLE
Make favourite/🖤 songs more accessible

### ISSUE THREAD

**Issue by metalheim** [navidrome/navidrome#1067]
### Is your feature request related to a problem? Please describe.

I want to access my favorited/hearted (🖤) songs more accessible, similar to the way streaming services like Spotify, Deezer, and alike handle it.
Currently, to find my favorite songs, I have to go to "Songs", then "filter by 🖤" and "sort by 🖤" to see my latest songs. 
The problem is: If I want to play all my favorite songs, I cannot. When I click on a song in that filtered Song-List, it only adds one song to the playQueue. I can only manually select one page of songs and add them to the playQueue.

### Describe the solution you'd like

A dedicated page for favorite/hearted (🖤) songs, that behaves more like a playlist (click one song and all get added to the playQueue). 
Ideally, this would be added to the side menu for quick accessibility (maybe only if `ND_ENABLEFAVOURITES` is set to true?)

### Describe alternative solutions that would also satisfy this problem

1. Change the songList behavior, so that all songs (considering current filters/sorting)´get added to the playQueue
2. Having a `smartPlaylist` that automatically adds favorited/hearted (🖤) songs.

### Additional context

Spinning the idea further, it's generally good UX practice to have a dedicated page for "personal" stuff, something that other solutions would call a "profile page", or similar. 
The distinction between data, that 

- "you" (aka the current user) put in/created
- data that is "just there", aka the admin added to the server

makes a user feel more at home.

That page could show an overview of the following "personal" data:

- Most played albums/songs
- favorited albums
- favorited songs
- favorited artists (once there is an artist page with the option to favorite an artist #961 )
- playlists you made

**Comment by RitikPandey1** [comment]
Yes, separate page for favourite songs would be nice. I'm working on it, soon gonna open PR.

**Comment by koehntopp** [comment]
+1

My preferred solution would be to have the exact same items that are there under the "Albums" menu, they're equally useful for songs.

And Kudos for being able to favourite albums - not many players can do that.

**Comment by abescalamis** [comment]
+1 for this why don't you use the same favourites tab for songs, artists and albums and it can be implemented in so many ways look at the example I did.

![MOCKUP](https://user-images.githubusercontent.com/36573279/135702843-3f14975e-6346-42da-8cb2-d9a447741c1d.jpg)

**Comment by deluan** [comment]
Hey @metalheim, I plan to revamp menus, views and filters in Navidrome UI, so I may implement this in the future.

For now, in the next version we will have the first version of Smart Playlists (#1417), and you will be able to create one for your Favourite songs with a `.nsp` file like this:

```json
{
    "any": [
        {"is": {"loved": true}}
    ],
    "sort": "title"
}
```

Let me know if you are ok with this solution, knowing that more will be done in the future, so I can close this issue.

**Comment by megamorf** [comment]
Any news on this, as it is now it's still only showing favourited albums. There's no way to have a unified favourites view at this point, right?

**Comment by deluan** [comment]
@megamorf As described in a comment above, there's now a way to have a "Favourites Songs" in the sidebar, using smart playlists.

There is no plans to implement a "unified" view for Favourites in the current UI.

**Comment by megamorf** [comment]
> @megamorf As described in a comment above, there's now a way to have a "Favourites Songs" in the sidebar, using smart playlists.
> 
> There is no plans to implement a "unified" view for Favourites in the current UI.

Thank you for the quick response and for the clarification @deluan! Ah, I see. I was under the impression that the Smart Playslists feature was not yet implemented.

**Comment by beyondlwm** [comment]
> > @megamorf As described in a comment above, there's now a way to have a "Favourites Songs" in the sidebar, using smart playlists.
> > There is no plans to implement a "unified" view for Favourites in the current UI.
> 
> Thank you for the quick response and for the clarification @deluan! Ah, I see. I was under the impression that the Smart Playlists feature was not yet implemented.

Hi megamorf, thanks for your hard work and I really love it.
It's not a push but I still wonder in which version we can have a way to show favorite songs? probably with "smart playlist" feature?

**Comment by metalheim** [comment]
> [..] I still wonder in which version we can have a way to show favorite songs? probably with "smart playlist" feature?

You can do it right now. At least if you're the admin user. 
Just create an empty text file somewhere in your music folder, name it "Favorites.nsp" and paste this in it:

`{{
  "any": [
    {"is": {"loved": true}}
  ],
  "sort": "dateLoved",
  "order": "desc"
}`

This will create a smart playlist named "Favorites" that appears in the playlist-list pane on the left and will automatically update itself with all your favorites.
